app/javascript/components/search/results/StudySearchResult.js

```diff
@@ -128,31 +129,52 @@ function facetMatchBadges(study) {
   </>)
 }
 
-/* displays a brief summary of a study, with a link to the study page */
-export default function SCPStudy({ study }) {
-  const termMatches = study.term_matches
-  const studyTitle= highlightText(study.name, termMatches).styledText
-  const studyDescription = formatDescription(study.description, termMatches)
-  const displayStudyTitle = { __html: studyTitle }
+/** Generate a cell count badge for SCP studies */
+function cellCountBadge(study) {
+  if (study.study_source === 'SCP') {
+    return <span className="badge cell-count">
+      {study.cell_count} Cells
+    </span>
+  }
+}
 
-  let inferredBadge = <></>
+/** Generate the inferredBadge for SCP studies */
+function inferredBadge(study, termMatches) {
   if (study.inferred_match) {
     const helpText = `${termMatches.join(', ')} was not found in study metadata,
      only in study title or description`
-    inferredBadge = <span className="badge soft-badge" data-toggle="tooltip" title={helpText}>text match only</span>
+    return <span className="badge soft-badge match-badge" data-toggle="tooltip" title={helpText}>text match only</span>
```

Comment:
do we need the extra match-badge class?  I don't see soft-badge used anywhere else, so maybe just consolidate those styles for now?